phpBB/includes/acp/acp_main.php

```diff
@@ -502,26 +502,8 @@ function main($id, $mode)
 
 		$upload_dir_size = get_formatted_filesize($config['upload_dir_size']);
 
-		$avatar_dir_size = 0;
-
-		if ($avatar_dir = @opendir($phpbb_root_path . $config['avatar_path']))
-		{
-			while (($file = readdir($avatar_dir)) !== false)
-			{
-				if ($file[0] != '.' && $file != 'CVS' && strpos($file, 'index.') === false)
-				{
-					$avatar_dir_size += filesize($phpbb_root_path . $config['avatar_path'] . '/' . $file);
-				}
-			}
-			closedir($avatar_dir);
-
-			$avatar_dir_size = get_formatted_filesize($avatar_dir_size);
-		}
-		else
-		{
-			// Couldn't open Avatar dir.
-			$avatar_dir_size = $user->lang['NOT_AVAILABLE'];
-		}
+		// Couldn't open Avatar dir.
+		$avatar_dir_size = $user->lang['NOT_AVAILABLE'];
```

Comment:
Actually it may be possible for some providers and I think it can still be useful (even if it displays the global usage).
You could add a get_total_usage or something like that in the provider which will return an array containing the current total usage and the maximum allowed usage (if it exists). If one information is missing its value being replaced by false.

Example: (all numbers are in bytes)
return ['usage' => 9999, 'max' => 10000] 
return ['usage' => 9999, 'max' => false]
return ['usage' => false, 'max' => false]